viewing_party/party.py

```diff
@@ -1,23 +1,188 @@
 # ------------- WAVE 1 --------------------
+# test_wave_01 - line 7 to line 59
+from logging import StringTemplateStyle
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    new_movie = {'title':title,
+    'genre': genre,
+    'rating': rating}
+    return new_movie
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
+# test_wave_01 - line 61 to line 81
+def add_to_watched(user_data,movie):
+    user_data['watched'].append(movie)
+    return user_data
 
+# test_wave_01 line 104 to line 121  
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+# test_wave_01
+def watch_movie(user_data,title):
+    
+    watch_list = user_data['watchlist']
+    for index in range(len(watch_list)):
+        movie = watch_list[index]
+        if movie['title']==title:
+            watch_list.pop(index)
+            user_data['watched'].append(movie)

```

Comment:
This works! But I think we can make it a little easier to read. We can also get rid of the `movie` variable on line 27 altogether by changing the for loop from `for i in range` to `for movie in watch_list` or `for movie in user_data['watchlist']`

```suggestion
    for movie in user_data['watchlist']:
        if movie['title']==title:
            user_data['watched'].append(movie)
            watch_list.remove(movie)
```
If we use `for movie in user_data['watchlist']`, we don't have an index position anymore, so we can't use `pop`. Here is an article about the difference between `remove` and `pop`

[https://www.geeksforgeeks.org/what-is-difference-between-del-remove-and-pop-on-python-lists/](https://www.geeksforgeeks.org/what-is-difference-between-del-remove-and-pop-on-python-lists/)